usecase/call_rpc.go

```diff
@@ -416,6 +429,44 @@ func (m *dependencyManager) CallRPC(ctx context.Context, w io.Writer, rpcName st
 	}
 }
 
+// Gets a request with the body containing the payload of its previous RPC
+// Only RPCs that are repeatable by definition will have their previous requests returned.
+func (m *dependencyManager) getPreviousRPCRequest(rpcName string, req interface{}) error {
+	id := rpcIdentifier(rpcName)
+	if m.state.rpcCallState[id].repeatable {
+		previousReqBytes := m.state.rpcCallState[id].reqPayload
+		if previousReqBytes == nil {
+			return errors.New(fmt.Sprintf("No previous request body exists for RPC: %s. Please issue a normal request.", rpcName))
+		}
+		message := req.(*dynamic.Message)
+		err := message.Unmarshal(previousReqBytes)
+		if err != nil {
+			return errors.Wrap(err, "Error while unmarshalling request for RPC [%s]. Please run without the -r option")
+		}
+		return nil
+	}
+	return errors.New("cannot rerun previous RPC as client streaming RPCs are not supported")
```

Comment:
There is no previous command, Evans shows this message. I think it's not expected behavior.
Should we show `No previous request body exists for RPC~` instead of it?

```
$ evans -s -r repl

api.Example@127.0.0.1:50051> call --repeat Unary
command call: cannot rerun previous RPC as client streaming RPCs are not supported 
```

Reply:
That's absolutely right. I missed this branch.